README.md

```diff
@@ -56,11 +117,50 @@ Supported OS versions:
   - YASM Assembler version 1.2.0 or later
 
 - __Build Instructions__
-  - `cd Build/linux`
-  - `./build.sh <release | debug>`
+  - As SVT-JPEG-XS is a regular CMake-build C/ASM package, the guide here targets
+    a first-time Linux developers and assumes you have a Windows computer with a fresh
+    installation of Windows Subsystem for Linux (WSL2).
+  - __Ubuntu distro installation__:
+    If you have an Ubuntu menu item in your start menu, you can proceed to the next item. 
+    If not, run 
+    ```
+    wsl --install
```

Comment:
Please make WSL related steps optional, they can be confusing for pure linux users